scripts/contentScript.js

```diff
@@ -639,10 +672,11 @@ const orchestrator = (tabUrl, websiteUrl, userId) => {
 						"repo_owner": ownerName,
 						"repo_name": repoName,
 						"user_id": userId,
-						"is_github": false
+						"repo_provider": "bitbucket"
 					}
-					const url = `${backendUrl}/relevance/pr`;
-					const highlightedPRIds = await apiCall(url, body);
+					const url = `${websiteUrl}/api/extension/relevant`;
+					const query_params = {"type": "review"};
```

Comment:
```suggestion
					const query_params = {type: "review"};
```
JS doesn't require the quotes like Python does.

Also, it would have been great if you could have defined an enum for the possible `type`s here. LOGAF: 1/10